test/TEST-16-EXTEND-TIMEOUT/assess.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+set -v -x
+
+# enough time for testsuite-success to complete
+sleep 18
+
+exit=0
+
+rm -f /test.log
+
+# This checks all stages, start, runtime and stop, can be extended by
+# EXTEND_TIMEOUT=1
+if [[ ! -f /success_all.success ]]
+then
+  journalctl -u testsuite-success-all.service >> /test.log
```

Comment:
We seem to use 4 spaces of indentiation in `test/*/test.sh`. Please make it the same here for consistency.